package.json

```diff
@@ -17,7 +17,7 @@
     "url": "https://github.com/NomicFoundation/hardhat-vscode/issues"
   },
   "scripts": {
-    "postinstall": "npm install --no-save --ignore-scripts --force @nomicfoundation/solidity-analyzer-win32-ia32-msvc@0.1.1",
+    "postinstall": "npm install --no-save --ignore-scripts --force @nomicfoundation/solidity-analyzer-win32-ia32-msvc@0.1.1 @nomicfoundation/slang-win32-ia32-msvc@0.10.1",
```

Comment:
not sure I understand why are we installing this here, rather than transitively through the child package deps?

Reply:
I have to say it is fuzzy. The git blame on that line reveals:

> To support the CI on testing an ugly hack is needed to force install the
windows ia32 version of the solidity analyzer. This is the architecture
that vscode extension host claims to run under, though the windows arch
according to node will be x64. This hack is in the root package.json as
a `postinstall` script.

I suspect that there is a clash between the architecture as exposed by node when run within the vscode extension host in CI tests and when the npm install is happening.

Do we need the slang dep added here? Not sure, if we remove it and the CI passes tests I think we are fine - though we should test the slang features on windows!

Reply:
I'll test if it works without adding this to the postinstall

Reply:
Thanks for explaining!

If we need it here, I suggest adding this info as a comment above. For example:

```json
{
  "_scripts_comment_": [
    "To support the CI on testing an ugly hack is needed to force install the",
    "windows ia32 version of the solidity analyzer. This is the architecture",
    "that vscode extension host claims to run under, though the windows arch",
    "according to node will be x64. This hack is in the root package.json as",
    "a postinstall script.",
  ],
  "scripts": {
    "postinstall": "npm install --no-save --ignore-scripts --force @nomicfoundation/solidity-analyzer-win32-ia32-msvc@0.1.1 @nomicfoundation/slang-win32-ia32-msvc@0.10.1"
  }
}
```

Reply:
Looks like tests are passing without the postinstall script :tada: